mozci/console/commands/push.py

```diff
@@ -280,14 +281,20 @@ def handle(self) -> None:
 
         self.errors = {}
         self.classifications = {}
+        self.regressions = {}
```

Comment:
Nit: let's call this `failures` instead of `regressions`, since e.g. the intermittent ones are not regressions